common/source/docs/common-buttons.rst

```diff
@@ -4,7 +4,7 @@
 Buttons
 =======
 
-Up to four external buttons or switches connected to the autopilot can be configured to trigger :ref:`Auxiliary Functions <common-auxiliary-functions>` similar to how RC channel switches can trigger them. These button inputs can be configured to use either analog voltage inputs (pull-to-ground or pull-to-high) or PWM signal inputs.
+Up to four external buttons or switches connected to the autopilot can be configured to trigger :ref:`Auxiliary Functions <common-auxiliary-functions>` similar to how RC channel switches can trigger them. These button inputs can be configured to use either digital logic signal analog voltage inputs (pull-to-ground or pull-to-high) or PWM signal inputs.
```

Comment:
Ah you're right here.. it's digital inputs not analog as I've said.  maybe we should remove the word "analog"?  Anyway, I'm happy either way.